plugins/source/aws/resources/services/apigateway/rest_api_authorizers.go

```diff
@@ -19,6 +19,9 @@ func RestApiAuthorizers() *schema.Table {
 				Name:     "account_id",
 				Type:     schema.TypeString,
 				Resolver: client.ResolveAWSAccount,
+				CreationOptions: schema.ColumnCreationOptions{
+					PrimaryKey: true,
+				},
 			},
 			{
 				Name:     "region",
```

Comment:
I think region should probably also be part of the PK, because we use the account-region multiplexer

Reply:
Oh no, it's part of the ARN, nevermind